hw2/04-background.js

```diff
@@ -1 +1,50 @@
 // Add your code here
+const btn = document.getElementById('control');
+let intervalTime = document.getElementById('intervalTime');
+let nIntervId;
+//function from:
+//https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+function getRandomInt(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min) + min);
+}
+
+function changeColor() {
+  const background = document.querySelector('body');
+  const red = getRandomInt(0, 255);
```

Comment:
not a big deal but I think your solution for getting a random int could be simpler, eg: 

const red = Math.floor(Math.random() * 255) 

this gives you an int between 0 and 255